ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -0,0 +1,576 @@
+        export OS_AUTH_URL="$(get_clouds_param 'auth_url')"
+        export OS_PROJECT_ID="$(get_clouds_param 'project_id')"
+        export OS_PROJECT_NAME="$(get_clouds_param 'project_name')"
+        export OS_USERNAME="$(get_clouds_param 'username')"
+        export OS_PASSWORD="$(get_clouds_param 'password')"
+        export OS_REGION_NAME="$(get_clouds_param 'region_name')"
+        export OS_ENDPOINT_TYPE="$(get_clouds_param 'interface')"
+        export OS_IDENTITY_API_VERSION="$(get_clouds_param 'identity_api_version')"
+        export OS_USER_DOMAIN_NAME="$(get_clouds_param 'user_domain_name')"
+        if [[ "$XTRACE_ENABLED" == 1 ]]; then
+          set -x
+        fi
+
+        mkdir -p /tmp/output
+        cd /tmp/output
+
+        function run-upgrade-tests() {
+          openshift-tests run-upgrade "${TEST_SUITE}" --to-image "${RELEASE_IMAGE_LATEST}" \
+            --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          return 0
+        }
+
+        function run-tests() {
+          openshift-tests run "${TEST_SUITE}" \
+            --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          return 0
+        }
+
+        function run-minimal-tests() {
+          # Only execute Smoke tests while the test infrastructure is getting
+          # prepared to run the actual suite reliably.
+          openshift-tests run openshift/conformance/parallel --dry-run |
+            grep 'Smoke' |
+          openshift-tests run -o /tmp/artifacts/e2e.log \
+            --junit-dir /tmp/artifacts/junit -f -
+          return 0
+        }
+
+        function run-no-tests() {
+          # This can be used if we just want to check the installer exits 0
+          echo "WARNING: No tests were run against the installed cluster"
+          return 0
+        }
+
+        ${TEST_COMMAND}
+
+    # Runs an install
+    - name: setup
+      image: ${IMAGE_OPENSTACK_INSTALLER}
+      terminationMessagePolicy: FallbackToLogsOnError
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: BASE_DOMAIN
+        value: ${BASE_DOMAIN}
+      - name: SSH_PUB_KEY_PATH
+        value: /etc/openshift-installer/ssh-publickey
+      - name: SSH_PRIV_KEY_PATH
+        value: /etc/openshift-installer/ssh-privatekey
+      - name: PULL_SECRET_PATH
+        value: /etc/openshift-installer/pull-secret
+      - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+        value: ${RELEASE_IMAGE_LATEST}
+      - name: OPENSHIFT_INSTALL_INVOKER
+        value: openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+      - name: OPENSTACK_IMAGE
+        value: rhcos
```

Comment:
This variable isn't used and should be deleted. Starting from 4.3, the installer knows how to download the right RHCOS image.